### PR TITLE
Milestone 1: Create Animal Table

### PR DESCRIPTION
**Pull request by binyamolango**
# Hi Reviewer 👋

> Here are the changes I implemented to the project

- Used [this template](https://github.com/microverseinc/curriculum-template-databases) to generate your repo.
- Created a database named vet_clinic.
- Created a table named Animals with the following columns: id: integer, name: string, date_of_birth: date, escape_attempts: integer, neutered: boolean, and weight_kg: decimal
- Copied the SQL I used in creating the animal table into the schema.sql file in the generated repository.
- Inserted animal details as a row.
- Copied the SQL I used in inserting into the data.sql file in the generated repository.
- Wrote queries for the different quests.
- Copied the SQL I used in writing queries into the queries.sql file in the generated repository.

Screenshot 2023-10-03 222743
Screenshot 2023-10-03 222836

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ Example command:
 
 ### Setup
 
-Clone this repository to your desired folder:
+Clone this repository to your desired folder: `git clone https://github.com/binyamolango/vet_clinic`
```
Comment (MahmoudBakr23) [inline README.md:109]: - [ ] In the README file please implement the following changes for a professional repo. 🙏 
1. Kindly remove the broken live demo link as it's not necessary for this milestone.
2. Kindly fill out the empty sections with project-related information.
3. Kindly fix the broken MIT link.

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @binyamolango 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done on creating the correct SQL statements for this project
- Descriptive PR summary

However, there is always room for improvement.

> I'd like to remind you that your long hours of work are appreciated. However, the code review process is for improving your code, not criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers, and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by binyamolango** [comment]
@MahmoudBakr23 Thank you for reviewing my project. I implemented all of the required changes.

**Review by fobadara — APPROVED** [review]
Hi @binyamolango, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

APPRVED
## Highlights
✅ Descriptive README
✅ Descriptive PR summary
✅ Clean code


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**